dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/organisationunit/OrganisationUnitController.java

```diff
@@ -212,16 +211,18 @@ else if ( anyQueryPropertySet )
         // Collection member count in hierarchy handling
         // ---------------------------------------------------------------------
 
-        IdentifiableObject member;
-
-        if ( memberObject != null && memberCollection != null && (member = manager.find( memberObject )) != null )
+        if ( memberObject != null && memberCollection != null )
         {
-            for ( OrganisationUnit unit : list )
+            Optional<? extends IdentifiableObject> member = manager.find( memberObject );
+            if ( member.isPresent() )
             {
-                Long count = organisationUnitService.getOrganisationUnitHierarchyMemberCount( unit, member,
-                    memberCollection );
+                for ( OrganisationUnit unit : list )
+                {
+                    Long count = organisationUnitService.getOrganisationUnitHierarchyMemberCount( unit, member.get(),
```

Comment:
Not part of this PR but just to highlight this method that just add complexity.
Why return a Long if then the null value is evaluated 0, instead of just returning a long with default value 0?